src/pages/home/home.ts

```diff
@@ -65,4 +65,10 @@ export class HomePage implements OnInit, OnDestroy {
       channelId: channelId
     });
   }
+
+  pushStreamingPage(url: string) {
```

Comment:
why don't use openPage('StreamingPage', this.settings) instead of pushStreamingPage. 
Same observation is valid for VideoPage.
Should simplify code

Reply:
use openPage passing params